components/dashboard/src/components/UsageView.tsx

```diff
@@ -246,6 +249,15 @@ function UsageView({ attributionId }: UsageViewProps) {
                                                     {usagePage.data?.creditsUsed.toLocaleString()}
                                                 </span>
                                             </div>
+                                            {currentOrg.data && currentOrg.data.isOwner && (
+                                                <div className="flex text-xs text-gray-600">
+                                                    <span className="dark:text-gray-500 text-gray-400">
+                                                        <Link to="/billing" className="gp-link">
+                                                            View Billing →
+                                                        </Link>
+                                                    </span>
+                                                </div>
+                                            )}
```

Comment:
**thought**: This will help make billing more visible until further billing UX improvements land from UX-23.